### DISCUSSION TITLE
[Projects Beta] Japanese input in task list is repeatedly entered in duplicate.

### DISCUSSION THREAD

**Discussion by shogo452** [community/community#19420] [Projects and Issues] [answered]
Hello, I found an issue with Japanese input in task list.

When I input Japanese in task list and a line break occurs, the same content is sometimes repeatedly entered in duplicate as shown in the GIF image below.

Thanks.

(For example in GIF image below, it happened when I enter "ううううう" and "えええええ".)
![chrome-capture-2022-5-25](https://user-images.githubusercontent.com/54160947/175773814-6b12e526-3702-4777-8daa-a75b6f9a183c.gif)

**Comment by melvincwng** [comment, marked as answer]
@shogo452, thank you for highlighting this issue to the community.

For bugs, please raise a ticket and kindly report to [Github Bug Report](https://support.github.com/contact/bug-report).

Hope this clarifies.

If this reply has helped you, please 'Mark as answer' so that the rest are aware this issue has been addressed.

Thank you and have a great day ahead 😊

**Reply by shogo452** [reply]
@melvincwng, thank you for your comment. I will raise a ticket about this issue to Bug report.
Thanks, 😊

**Reply by melvincwng** [reply]
@shogo452, Hi Shogo no problem! Pleasure to have helped you.

ありがとう 😄

**Comment by sainu** [comment]
This bug has been fixed now.
